src/main/java/io/github/darealturtywurty/turtychemistry/client/util/screens/FoundryScreen.java

```diff
@@ -0,0 +1,83 @@
+package io.github.darealturtywurty.turtychemistry.client.util.screens;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import io.github.darealturtywurty.turtychemistry.TurtyChemistry;
+import io.github.darealturtywurty.turtychemistry.block.entity.FoundryBlockEntity;
+import io.github.darealturtywurty.turtychemistry.menu.FoundryMenu;
+import io.github.darealturtywurty.turtylib.client.ui.components.FluidWidget;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.level.material.Fluids;
+import net.minecraftforge.fluids.FluidStack;
+import org.jetbrains.annotations.NotNull;
+
+public final class FoundryScreen extends AbstractContainerScreen<FoundryMenu> {
+    public static final ResourceLocation TEXTURE = new ResourceLocation(TurtyChemistry.MODID,
+            "textures/gui/clay_alloy_furnace.png");
+    private FluidWidget fluidWidget;
+
+    public FoundryScreen(final FoundryMenu screenContainer, final Inventory inv, final Component title) {
+        super(screenContainer, inv, title);
+        this.leftPos = 0;
+        this.topPos = 0;
+        this.imageWidth = 176;
+        this.imageHeight = 166;
+    }
+
+    @Override
+    protected void init() {
+        super.init();
+        this.fluidWidget = addRenderableOnly(
+                new FluidWidget(new FluidStack(Fluids.LAVA, this.menu.getData().get(2)), 1920 >> 4, 1080 >> 4, 9, 9, 9,
```

Comment:
Please stop using `>>`. I know you are going to say its an optimization. But yes. It's a micro-optimization at the cost of code readability. Code readability you need to understand is more important when working on a team then things like micro-optimizations are. The amount of time this will strip away is maybe. 5ns?

That is at the cost of a lot of code readability as not everyone (including me) is familiar with bit shifting as a replacement to divisions. It doesn't help here because these are also very arbitrary values, so I'm not even sure what these numbers are supposed to represent.